### ISSUE TITLE
Allow Authors to withdraw papers at any time

### ISSUE THREAD

**Issue by jmacgreg** [pkp/pkp-lib#1813]
Authors should be able to withdraw papers at any time prior to final publication. Authors can currently only withdraw/delete papers that are incompletely submitted. Some questions: 

- is there a specific cutoff for when an author can or cannot withdraw a paper without further intervention/steps taken - eg. after publication? 

- what would the procedure be? A "Withdraw" button and an associated email to the editor responsible?

**Comment by asmecher** [comment]
Interested in @willinsky's thoughts on this.

**Comment by t4x0n** [comment]
In the editorial process, you not should withdraw your paper at any time, make this part of the process so easy is not a good idea (I say this as editor and author), is important at the first part of the process but if this manuscript was reviewed and improved for publishing in that journal? withdraw the submission is anti-ethical and the reasons should be very justified.

**Comment by amandastevens** [comment]
What about if the author still has to contact the editor to request that the submissions be withdrawn but there is an option for the editor to record the decision as "withdrawn" rather than "rejected."

**Comment by alexxxmendonca** [comment]
I agree with @t4x0n and @amandastevens.

I don't think the authors should have full liberty to withdrawn their own submissions. They should be able to request it and the ones to actually do it should be the journal.

**Comment by t4x0n** [comment]
I agree with you @alexxxmendonca but, if this will be implemented, when the author's making the request is important to obligate the author to say WHY they are requesting withdraw their submission, like an obligated text area. In my experience, withdraw a manuscript is not a common choice and normally it is accompanied by a problem with an editor or excessive delays in the review process, but is important explain that to the editor in chief, in some cases this is more like a warning and the editor in chief can make a move for accelerate the process and avoid the withdraw.

**Comment by alexxxmendonca** [comment]
I fully agree, @t4x0n.

**Comment by marcusrossberg** [comment]
Shouldn't the decision when an author is allowed to withdraw be left to the relevant editor? 
Not having the function available at all seems to me (speaking as an editor) to be a disadvantage.

**Comment by NateWr** [comment]
We've had a number of requests from the community for a withdrawn editorial decision that does not require recording a rejected decision. See the [forum](https://forum.pkp.sfu.ca/t/withdrawn-articles/48129/3). I've applied the Community Priority label and adjusted the description to reflect the discussion above.

**Comment by carzamora** [comment]
Yes, I agree with the Update:

> Update: 2020-08-04: It should be possible to withdraw a paper at any point during the editorial process without recording a rejected decision. Following the discussion below, it seems the preference is for the editor to have this control. The author can send a request to the editor

**Comment by willinsky** [comment]
I agree with everyone that the editor alone should be able to withdraw or retract a paper. I would suggest that we structure it in the following way (as this would potentially add another element to the Journal Integrity Initiative that I have been promoting for OJS): 

On the editorial workflow, we add another button  in the Submission stage and underneath Decline Submission called Withdraw Submission (black font on gray background) that will turn to Retract Publication after publication and remain active. 

(a) Prior to publication, clicking Withdraw Submission would call up a Withdrawal pop-up that would record whether this withdrawal of the submission (to be archived) was initiated by the editor or the author, and a reason given for taking this step. Once the submission is in the archives, it will be listed as “Withdrawn” and in the Submission stage will list who withdrew it and why.  

(b) After publication, the Withdraw button becomes Retract Submission and remains part of the workflow. Clicking it would call up a Retraction pop-up that would record whether the retraction of the submission (which will remain in the archive) was initiated by the editor or the author, and a reason given for the step (which will be included in the Submission stage), as well as include an additional box for Retraction Notice to be added to the article landing page, with the term “Retracted” added to its Table of Contents entry. Editors may wish to add this information to the galley so that the PDF and HTML are so labelled, but this is not something we can do for them.

**Comment by alexxxmendonca** [comment]
For OPS, ASAPBio has came up with a set of best practices and recommendations for handling with preprint withdrawal and removal at the **Defining preprint withdrawal & removal** section of this document: https://osf.io/8dn4w/

From what I could tell, it seems to be very similar to the journal approach, but I think it would be worth making sure that OPS is following these recommendations.

**Comment by willinsky** [comment]
Thanks @alexxxmendonca for the reference to the paper with the "withdrawal and removal" recommendations. My suggested principles above have been further edited, are now largely in accord with its recommendations and examples (they do say these are "not settled practices" but I went with what I found made the most sense and aligned with OJS.

**Comment by ajnyga** [comment]
Has someone discussed with Crossref on how withdrawal should affect the DOI or the metadata attached to the DOI?

Also, do we want the withdrawal to affect all versions of preprint by default? Can there be a situation where only the latest version of the preprint is withdrawn?

**Comment by asmecher** [comment]
@willinsky, I would be on-board with your proposal for OJS older than 3.2.x, but now that 3.2.x is introduced with versioning I'm not so sure we can rely on the Editorial Decision part of each workflow stage when the publishing action is elsewhere. The Publication area is now a separate tab from the Workflow and has publication status information:

![image](https://user-images.githubusercontent.com/200411/95387582-c43ead00-08a5-11eb-97f4-59c972376cd0.png)

I would expect retractions/withdrawals to happen here, rather than on the workflow tabs.

**Comment by NateWr** [comment]
Can I suggest that we separate out withdrawal (before publication/posting) and retraction (after publication/posting) into two separate issues/discussions? Retraction has a lot of implications that need to be handled carefully (notice on the landing page and galleys, update of metadata deposits and distribution, the ability to indicate a date and reason for retraction, etc), whereas withdrawal is comparatively straightforward.

This issue could focus on withdrawal and a new one could be created to discuss retraction.

**Comment by ajnyga** [comment]
Sounds like an important distinction.

Withdrawal would the be something you can in OJS/OMP/OPS as long as the submission is not published/posted online. The button could be situated under the decline button. In OPS the button should be available of authors as well? This I can implement fairly quickly to all three applications.

Retraction would the be something that happens to a published submission. The button should be under the Publication tab. Retraction would then affect the public landing page and metadata and basically create a tombstone? Would this be something that could be done to a single version of the submission? Do we have use cases for this feature?

**Comment by NateWr** [comment]
> Withdrawal would then be something you can do in OJS/OMP/OPS as long as the submission is not published/posted online. The button could be situated under the decline button. This I can implement fairly quickly to all three applications.

Let's not tie this to a particular stage decision. Instead, I'd like to think about where this might sit outside of the workflow. (It will also better prepare us for retractions.) 

For a while, I've been thinking about converting some of the buttons at the top into a dropdown menu. This will also give us an extensible pattern for other plugins to use as needed.

![withdraw-dropdown](https://user-images.githubusercontent.com/2306629/95454447-61601b00-0964-11eb-94f0-ffdb9fc104b2.png)

Clicking this button should open a modal to confirm the withdrawal and present an email for review, that would be sent to the author(s). The editor should have an opportunity to enter a reason for withdrawal (one-line text field).

When a paper is withdrawn, let's add a withdrawn badge at the top and hide the editorial decisions behind a prompt, similar to how we do with decisions that are completed on a stage.

![withdraw-decisions](https://user-images.githubusercontent.com/2306629/95454577-90768c80-0964-11eb-9ab4-d78216f0d11d.png)

Finally, the status will need to be reflected in the publications tab and the schedule for publication button should be removed.

![widthraw-publication](https://user-images.githubusercontent.com/2306629/95454631-9ec4a880-0964-11eb-87ec-2b6a108bf84d.png)

We will need a new `STATUS_WITHDRAWN` for this. Editorial stats might get a little complicated (acceptance/rejection rates need to make sure that they are excluding withdrawals from the divisor). And we'll need to make sure that these submissions are moved to the archived lists, and review assignments are cancelled or otherwise updated.

Here's my list of things that I think would need to happen:

1. UI additions as described.
2. New email template for withdrawal notice for authors.
3. New email template for withdrawal notice to reviewers, that would be sent to assigned reviewers with a pending review assignment.
4. On withdrawal, add an entry to the submission's activity log that includes the reason for withdrawal.
5. A new `STATUS_WITHDRAWN` constant and an API endpoint to `PUT /submissions/{id}/withdraw` (
6. Submission lists should load withdrawn submissions in the archive list.
7. When withdrawn, the "Withdraw Submission" button should change to "Revert Withdrawal". An event should be added to the activity log but no notifications sent to authors/reviewers. Editor will need to decide who to alert and whether to restart review assignments or make new assignments.

Now there are a number of more technical questions about how to perform this withdrawal action, and what impacts it has. I think that we should have a separate service method, `PKPSubmissionService::withdraw()`, that encapsulates all of the actions that need to happen when a withdrawal takes place.

Additionally, we will need some validation rules in place to prevent a withdrawal from happening outside of this service class. Probably a new API endpoint at `PUT /submissions/{id}/withdraw`. For example, it should not be possible to withdraw a submission when one or more publications are published or scheduled. The editor should be asked to unschedule the publication(s) before withdrawing.

The API endpoint handler for editing a submission, `PUT /submissions/{id}` should return an error if the user tries to edit the `status` to `STATUS_WITHDRAW`. The error message should instruct them to make the requerst to the withdraw endpoint. We will still allow this status change in the `PKPSubmissionService::edit()` method, but restricting the API provides some protections from accidental misuse of the API.

> Retraction...

Let's talk about that in a separate issue. I think from a UI perspective, taking the action could be similar to withdrawing it. But all of the other details will need to be worked out separately.

**Comment by alexxxmendonca** [comment]
I am not that confident that weithdawal should be restricted to pre-publication.

We've has cases of duplicate preprints being posted by accident (author confused and made a new submission instead of a new version). Both were posted but we needed to remove one of them. I am not sure this would be the case for a retraction. There is nothing ethically wrong for it to be "retracted".

**Comment by ajnyga** [comment]
Is that a case where the submission could be just deleted?

**Comment by alexxxmendonca** [comment]
It might have been accessed and linked and referenced so for consistency's sake, we need to keep the record but update the PDF.

This is what we did (check the PDF): https://preprints.scielo.org/index.php/scielo/preprint/view/547

**Comment by NateWr** [comment]
@alexxxmendonca you're right that withdrawal/retraction is going to be a different thing in OPS. Because there is little time between submission and publication, most withdrawals will need to occur after publication. And retraction isn't really a thing in preprints, is it?

From a technical perspective, withdrawing after posting a preprint in OPS will be very similar to a retraction in OJS/OMP. For this reason, I think that we should discuss and implement it in a separate issue that discusses retractions. Retractions (OJS/OMP) and after-posting withdrawals (OPS) are considerably more complicated because we must account for the fact that the item has been publicly distributed.

**Comment by willinsky** [comment]
Just to reinforce Nate’s point — and I’m happy to see withdrawal and retraction in separate issues -- the important point here is that we want to be clear that papers are not published in OPS, they are posted or shared or circulated. This has become increasingly important  with Covid-19, in which the distinctions between a work that has been published (with peer review) and which circulated in advance of publication have become part of media take-up and public awareness.

As a result, to apply the general journal standards to this distinction, papers posted in OPS can be withdrawn (and thus “retraction” per se, does not apply). We are upping the game, with regard to the integrity, by asking that “withdrawal” be made public with an explanation.

Thanks
John

On Oct 8, 2020, at 6:32 AM, Nate Wright <notifications@github.com<mailto:notifications@github.com>> wrote:



@alexxxmendonca<https://github.com/alexxxmendonca> you're right that withdrawal/retraction is going to be a different thing in OPS. Because there is little time between submission and publication, most withdrawals will need to occur after publication. And retraction isn't really a thing in preprints, is it?

From a technical perspective, withdrawing after posting a preprint in OPS will be very similar to a retraction in OJS/OMP. For this reason, I think that we should discuss and implement it in a separate issue that discusses retractions. Retractions (OJS/OMP) and after-posting withdrawals (OPS) are considerably more complicated because we must account for the fact that the item has been publicly distributed.

**Comment by alexxxmendonca** [comment]
> And retraction isn't really a thing in preprints, is it?

Retraction is more associated with peer-review, indeed.

I very much agree with @willinsky's [comment](https://github.com/pkp/pkp-lib/issues/1813#issuecomment-705578602).

While we are discussing this, I'd recommend checking the section **Defining preprint withdrawal & removal** of ASAPBIo's _[Building trust in preprints: recommendations for servers and other stakeholders](https://osf.io/8dn4w/)_ report for suggestions on how we can deal with this.

They recommend "Withdrawn" and "Removal". "Removal" seems to be used in much more serious cases (Serious legal issues, erious risk to public health or potential public security concerns).

**Comment by NateWr** [comment]
Thanks @alexxxmendonca! That's precisely the information we need, and it even has examples! It's clear now that withdrawal/retraction/removal in OPS will be very different from OJS/OMP. I've created a new issue based on the recommendations in this document and we can continue discussion there: https://github.com/pkp/pkp-lib/issues/6271.

**Comment by willinsky** [comment]
Agreeing with Nate's "More" dropdown approach (as well as status indicators), which makes particular sense for withdrawal/retraction due to the rarity of the event and as the More dropdown remains in place throughout the workflow and after publication. I think that the two can be treated as a pair rather than separately. Here's one way it might be handled:

More
 /\ 
 * Activity Log 
 * Library
 * Remove item 

"Remove item" leads to....

Remove Item

( ) Withdraw submission, which takes place prior to publication (and also applies to posted preprints).  
( ) Retract publication, which takes place following publication.

Explain the reason for and instigator for this move, which for the withdrawal will be dated and added to the submission metadata under the title Withdrawn and the submission archived, or for the retraction will be dated and posted on the article landing page as a "Retraction Notice" with "Retracted" added to the Table of Contents (while the editors should add this notice and explanation to the publication itself).

**Comment by promedacademy** [comment]
I think authors have the right to request to withdraw their articles at any time before publication, this is  because some journals cannot fulfill their promises and presence of withdrawal button increase the trust of the authors towards the journal and that it is not a predator journal  the final decision to accept the withdrawal request might be assigned to different users roles from section editors , journal manager, or editor-in-chief

**Comment by suenugus** [comment]
I agree with promedacademy that authors should be able to withdrawn submissions. In any event the ability for someone to do this is essential for all sorts of reasons. I hope the priority tag on this will have this enabled soon.

**Comment by alexxxmendonca** [comment]
Pushing this for OPS as well.

We are currently facing a situation where an author has systematically submitted the same made-up manuscript even after we rejected it. The manuscript is clearly made-up (it's made of mostly nonsense) and the author has submitted 8 times the same manuscript, 5 of which we rejected.

Right now, these submissions are _clogging_ our submission queue. We don't have any other choice but to reject and this is affecting our rejection rate.

Having a way to withdrawn the papers would allow us to deal with these cases without affecting our editorial statistics.

**Comment by ajnyga** [comment]
This is how Withdrawal, Retraction and Removal is handled in OJS/OPS/OMP. Note that the central difference here is that Withdrawal means different things in preprints and articles as it has been shown in the discussion above. 

| stage            | system level     | OJS label | OMP label | OPS label | DOI                                                                 | Files                   | TOC/Catalogue              |
|------------------|------------------|-----------|-----------|-----------|---------------------------------------------------------------------|-------------------------|----------------------------|
| submitted        | STATUS_CANCELED  | Withdrawn | Withdrawn | Canceled  | no registered DOI, no problems                                      |                         |                            |
| published/posted | STATUS_RETRACTED | Retracted | Retracted | Withdrawn | Landing page with retraction note                                   | Files still accessible? | Marked as retracted in TOC |
| published/posted | STATUS_REMOVED   | Removed   | Removed   | Removed   | Landing page with retraction note (we still need this for the DOI!) | Files not accessible    | Not in TOC or search       |

**Comment by pmangahis** [comment]
+1 from another hosted clint that would be interested in this

**Comment by NateWr** [comment]
Thanks @ajnyga, that's a really helpful summary. I agree with everything you posted. More comments:

> If the submission has been public even at some point, we should block it.

Do you have something in mind to determine this? I can only think of checking the activity log for a `SubmissionEventLogEntry::SUBMISSION_LOG_METADATA_PUBLISH` entry. We should probably back this up with a check of the submission's status, which will be published if any publication is published.

> Should we allow taking back the decision? Definitely...

We have a bit of a naming pattern you can follow with the `Decline` and `RevertDecline` decisions. When reverting the withdrawal, always change `STATUS_CANCELLED` to `STATUS_QUEUED`. If they want to publish or decline it after that, they can take another decision.

> Where do we want to place the button for the Decision?

I think the More dropdown I described in [this comment](https://github.com/pkp/pkp-lib/issues/1813#issuecomment-705525532) is my preference for now.

> Do we send an email about the Decision? ... yes, to all authors of the submission...

I think we need a few emails:

1. To the assigned author.
2. A copy of the email to all contributors. (See `DecisionNotifyOtherAuthors`)
3. An email to all assigned editorial staff (managers, subeditors, assistants).

> do we need to have this decision https://github.com/pkp/pkp-lib/issues/6982 separately

I don't think so? They seem different to me and can be implemented separately.

> Something else?

A few more things to note:

1. Submissions in the archived list should show a withdrawn badge instead of declined or published.
2. Show "Withdrawn" at the top of the submission workflow page, where Scheduled and Published appear next to the title.
3. Currently, the `status` of a submission can be edited through the API. I'm [planning to lock that down](https://github.com/pkp/pkp-lib/issues/7191#issuecomment-1171014633). But we should make sure that is in place when this work is completed, because we want to control who can change the status of a submission and when.
4. When you dig into the decisions, you'll find one hurdle is that every decision type has to declare what stage it appears in. You'll need a separate decision constant and class for withdrawal in each stage. Kind of an uncomfortable situation but just use a base class for now, and create a slim child class for each stage.

**Comment by ajnyga** [comment]
> Do you have something in mind to determine this? I can only think of checking the activity log for a SubmissionEventLogEntry::SUBMISSION_LOG_METADATA_PUBLISH entry. We should probably back this up with a check of the submission's status, which will be published if any publication is published.

The activity log entries are what I was thinking of. I think we discussed using those in some other issue at some point. But I can see no problem if we check both the activity log and current status. Or do you think that we would want that data saved in a more easily accessible way? I mean some marker in publication_settings or elsewhere saying that this publication has been published at some point? That might be useful also when looking at things like tombstones, retraction etc.

> I don't think so? They seem different to me and can be implemented separately.

Ok, I was thinking about this from the perspective of limiting the amount of decisions we have. But it could be that my thinking is influenced by the fact that adding new decisions used to be quite difficult. I was thinking that for an editor the most important thing is not to confuse these with actual rejected submissions. Not sure if many would like to see a difference between Withdrawn and just moved to archive for another reason. But of course in the long run better to have a status like STATUS_DELETED if that is what we are going for in that issue.

> When you dig into the decisions, you'll find one hurdle is that every decision type has to declare what stage it appears in. You'll need a separate decision constant and class for withdrawal in each stage. Kind of an uncomfortable situation but just use a base class for now, and create a slim child class for each stage.

Ok, good to know!

**Comment by NateWr** [comment]
> Or do you think that we would want that data saved in a more easily accessible way? I mean some marker in publication_settings or elsewhere saying that this publication has been published at some point?

No, I don't think we should add anything in here just yet.

**Comment by bozana** [comment]
s. aslo https://groups.niso.org/higherlogic/ws/public/download/30869/NISO_RP-45-2024_CREC.pdf
